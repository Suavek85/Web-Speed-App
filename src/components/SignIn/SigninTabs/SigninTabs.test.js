import React from 'react'
import { shallow } from 'enzyme'
import SignInTabs from './SigninTabs'

describe('SignInTabs snapshot', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<SignInTabs />)
    expect(wrapper).toMatchSnapshot()
  })

})