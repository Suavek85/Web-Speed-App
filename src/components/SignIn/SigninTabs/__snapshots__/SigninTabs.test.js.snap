// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInTabs snapshot should match the snapshot 1`] = `
<div>
  <div
    className="headeritemWrapper headeritemWrapperInactive"
    id="signin"
  >
    <span>
      Sign in
    </span>
  </div>
  <div
    className="headeritemWrapper headeritemWrapperInactive"
    id="signup"
  >
    <span>
      New account
    </span>
  </div>
</div>
`;
